ui/src/segments/details/SegmentDetailsView.js

```diff
@@ -73,39 +77,36 @@ const SegmentDetailsView = ({ projectId, segmentId, ...props }) => {
           <Fragment>
             {!(props["*"] === "edit") ? (
```

Comment:
Similar comment to above: Could we do `{!(props["*"] === "edit") && (...)` here instead?